x/incentives/keeper/distribute_test.go

```diff
@@ -539,25 +539,20 @@ func (s *KeeperTestSuite) TestDistribute_ExternalIncentives_NoLock() {
 			// can function properly.
 			s.Ctx = s.Ctx.WithBlockTime(oneHourAfterDefault)
 
-			s.FundAcc(s.TestAccs[0], tc.gaugeCoins)
-
 			// Create gauge and get it from state
-			externalGaugeid, err := s.App.IncentivesKeeper.CreateGauge(s.Ctx, tc.isPerpertual, s.TestAccs[0], tc.gaugeCoins, tc.distrTo, tc.startTime, tc.numEpochsPaidOver, defaultCLPool)
-			s.Require().NoError(err)
-			externalGauge, err := s.App.IncentivesKeeper.GetGaugeByID(s.Ctx, externalGaugeid)
-			s.Require().NoError(err)
+			externalGauge := s.createGaugeNoRestrictions(tc.isPerpertual, tc.gaugeCoins, tc.distrTo, tc.startTime, tc.numEpochsPaidOver, defaultCLPool)
```

Comment:
Note: it was required to modify the test and create this helper due to stricter validation in `CreateGauge`. The goal is to setup an invalid gauge state so that `Distribute()` function below fails. With the old logic, it was impossible to hit edge cases while keeping stricter validation